src/modules/ROOT/pages/changelog/202209/02.json

```diff
@@ -0,0 +1,393 @@
+    {
+      "what": [
+        "Fixed translations for some resources"
+      ],
+      "how": [
+        "Fixing their corresponding file names into locale repo"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-locale/commit/b2ad8c2"
+      ]
+    },
+    {
+      "what": [
+        "Record search endpoints needlessly degrading performance by computing totals and paging cursors even when not required.",
+        "The performance hit was noticeable on larger datasets."
+      ],
+      "how": [
+        "Short-circuiting processing sooner when totals and page cursor navigation was not required.",
+        "This doesn't change external behavior as those parameters were not included even tho they were computed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/24b5d7c72dca66a71490c47bcbc50419ee53a79e"
+      ]
+    },
+    {
+      "what": [
+        "Data aggregation count crashing the server when no arguments were provided (`count()``).",
+        "The old version allowed this format while the new one did not and it crashed the server."
+      ],
+      "how": [
+        "By adjusting validations and adding additional exceptions for this specific case"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/fd3552eaa5cfad8d880a00ee046feec154578d18"
+      ]
+    },
+    {
+      "what": [
+        "Action log search results did not change if you reset the filter parameters."
+      ],
+      "how": [
+        "By fixing the update of search parameters in the URL."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/ca14671d2300d39699363a3b21b071032f230321"
+      ]
+    },
+    {
+      "what": [
+        "Geometry values not being saved if changed manually in inputs."
+      ],
+      "how": [
+        "Fixed reactivity of local value, by watching its changes with deep enabled.",
+        "In short the value was not being updated because Vue didn't watch for it being updated via inputs."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/893e644e1ae8cce7666fbf4280271e72684afa73"
+      ]
+    },
+    {
+      "what": [
+        "When using the ${userID} syntax in record export filter, it was not being evaluated with the current users ID."
+      ],
+      "how": [
+        "Added evaluatePrefilter to the makeFilter method inside Record/Exporter/Configurator.",
+        "Therefore `${userID} syntax is now properly evaluated."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/9e9898bcc7722e56acd34c5ff8cafb6a955de6b0"
+      ]
+    },
+    {
+      "what": [
+        "Privacy web app was not included into HTTP_WEBAPP_LIST option (environment variable)"
+      ],
+      "how": [
+        "Update cue definitions for options for HTTP server and changed the default value to include privacy."
```

Comment:
Can you tell here that the privacy was added to the webapp list by default (this outlines the solution better)